style.css

```diff
@@ -130,3 +130,416 @@ p {
 .menu-bar {
   display: none;
 }
+
+/* work section */
+.work {
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.work-section {
+  display: grid;
+  grid-template-columns: 100%;
+}
+
+.work-1 {
+  display: flex;
+  flex-direction: column;
+}
+
+.work1-visual {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.work1-visual > h1 {
+  font-family: "Crete Round", serif;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 52px;
+  letter-spacing: 0;
+  align-items: center;
+  color: #1c1a19;
+  margin-left: 10px;
+  margin-top: 80px;
+}
+
+.line {
+  border: 1px solid black;
+  margin: 0 34px 56px 34px;
```

Comment:
- [ ] Please adhere to the project requirement and *Figma* design closely and try to make it exactly the same as given in the design because it is very important to complete the project as per the requirement given by **Microverse** as a software developer which will help you to make you detail-oriented developer. Kindly remove the margin and add `width: 100%`. It will make your design similar to what has been given in the `Figma` design.

- Please use `border: thin solid black;` instead of 1px solid back border.

<table>
<tr>
<th align=“middle”>Your Design</th>
<th align=“middle”>Figma Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-31 16-13-25](https://github.com/sulaimanahmadi/Portfolio/assets/66476329/2e3680e8-f470-45e2-8317-015afcea5a65)

</td>
<td>

![image](https://github.com/sulaimanahmadi/Portfolio/assets/66476329/7a6c8dff-dd9e-4b97-a76f-3efdc2a72277)

</td>
</tr>
</table>